wire-ios-data-model/Source/ManagedObjectContext/Migration/CoreDataMessagingMigrationVersion.swift

```diff
@@ -83,8 +84,10 @@ enum CoreDataMessagingMigrationVersion: String, CoreDataMigrationVersion {
 
     var nextVersion: Self? {
         switch self {
-        case .v130:
+        case .v131:
             nil
+        case .v130:
+            .v131
         case .v129:
             .v130
```

Comment:
I think it may be possible to do this
```swift
case .v131:
    nil
case .v129, .v130:
    .v131
```
if the last version also targeted only develop